service/kvmconfig/v3/resource_set.go

```diff
@@ -35,10 +35,8 @@ type ResourceSetConfig struct {
 	Logger             micrologger.Logger
 	RandomkeysSearcher randomkeys.Interface
 
-	GuestUpdateEnabled    bool
-	HandledVersionBundles []string
```

Comment:
We do not need to configure this dynamically anymore, which means less human failure potential.